src/proposals/20240905/SparkEthereum_20240905.t.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import 'src/SparkTestBase.sol';
+
+import { IKillSwitchOracle } from 'lib/sparklend-kill-switch/src/interfaces/IKillSwitchOracle.sol';
+
+interface IOracle {
+    function latestAnswer() external view returns (int256);
+}
+
+contract SparkEthereum_20240905Test is SparkEthereumTestBase {
+
+    address internal constant WETH_ORACLE_OLD   =   0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419;
+    address internal constant WSTETH_ORACLE_OLD =   0x8B6851156023f4f5A66F68BEA80851c3D905Ac93;
+    address internal constant RETH_ORACLE_OLD   =   0x05225Cd708bCa9253789C1374e4337a019e99D56;
+    address internal constant WEETH_ORACLE_OLD  =   0x1A6BDB22b9d7a454D20EAf12DB55D6B5F058183D;
+
+    address internal constant WETH_ORACLE       =   0xf07ca0e66A798547E4CB3899EC592e1E99Ef6Cb3;
+    address internal constant WSTETH_ORACLE     =   0xf77e132799DBB0d83A4fB7df10DA04849340311A;
+    address internal constant RETH_ORACLE       =   0x11af58f13419fD3ce4d3A90372200c80Bc62f140;
+    address internal constant WEETH_ORACLE      =   0x28897036f8459bFBa886083dD6b4Ce4d2f14a57F;
+
+    address internal constant WBTC_BTC_ORACLE   =   0xfdFD9C85aD200c506Cf9e21F1FD8dd01932FBB23;
+
+    constructor() {
+        id = '20240905';
+    }
+
+    function setUp() public {
+        vm.createSelectFork(getChain('mainnet').rpcUrl, 20613541);  // Aug 26, 2024
+        payload = deployPayload();
+
+        loadPoolContext(poolAddressesProviderRegistry.getAddressesProvidersList()[0]);
+    }
+
+    function test_validateOracles() public {
+        vm.startPrank(Ethereum.AAVE_ORACLE);
+        // Less than 1% difference between the old and new oracles
+        assertApproxEqRel(IOracle(WETH_ORACLE_OLD).latestAnswer(),   IOracle(WETH_ORACLE).latestAnswer(),   0.01e18);
+        assertApproxEqRel(IOracle(WSTETH_ORACLE_OLD).latestAnswer(), IOracle(WSTETH_ORACLE).latestAnswer(), 0.01e18);
+        assertApproxEqRel(IOracle(RETH_ORACLE_OLD).latestAnswer(),   IOracle(RETH_ORACLE).latestAnswer(),   0.01e18);
+        assertApproxEqRel(IOracle(WEETH_ORACLE_OLD).latestAnswer(),  IOracle(WEETH_ORACLE).latestAnswer(),  0.01e18);
+        vm.stopPrank();
+    }
+
```

Comment:
Lets add a test specifically for Aggor configuration.

We should be asserting for all four Aggor contracts (can probably use a for loop these should be the same):

```
ageThreshold
agreementDistance
authed array
tolled array
chainlink address
chronicle address
decimals
latestAnswer
All uniswap related values
wards
```

Reply:
@0xShaito, do I understand correctly that `latestAnswer` check is in `test_validateOracles`?
Could you point to a `wards` check, please?

Reply:
Yes! latestAnswer is checked in the validate oracles and wards is the same as authed. The wards function tells you if an address is authed and the authed function lists all of the addresses:

https://etherscan.io/address/0x69115a2826Eb47FE9DFD1d5CA8D8642697c8b68A#code#F11#L43